source/config/config.go

```diff
@@ -68,9 +70,32 @@ func ParseSourceConfig(ctx context.Context, cfg map[string]string) (SourceConfig
 		)
 	}
 
+	var fields []string
+	if cfg[ConfigKeyFields] == "" {
+		fields = getOrderedFields([]string{})
```

Comment:
Why do the fields need to be ordered?

Reply:
coz, we need `Id` `createdAt` `updated` fields in order to create sdk.Record. If user doesn't add this while requesting the `fields` we cannot identify records uniquely. And we require these 3 fields in the list of fields. so that method will add these 3 fields if not present in the list of fields.

Reply:
How does ordering fit into that?

Reply:
code now is updated. there is no `getOrderedFields` function